src/utils.js

```diff
@@ -44,12 +69,12 @@ function runPulumiCommand(args, interactive = true, twilioClient) {
       Printer.printHeader('Pulumi CLI output');
       childProcess.execFileSync('pulumi', args, {
         stdio: 'inherit',
-        env: getEnvironmentVariables(twilioClient),
+        env: getEnvironmentVariables(twilioClient, args),
```

Comment:
I'm wondering if there's a better way of avoiding the loop than passing the entire `args`. I'm wondering if we should handle that within the `runPulumiCommand()` and keep `getEnvironmentVariables()` abstract. We could use the second argument as a boolean that skip reading the `.env` file if `false`. We then decide in the `runPulumiCommand()` if we want to run that or not depending on the command.  What do you think @kaiquelupo @bermudezmt ?

Reply:
Hey, I liked your proposal. I am sending a commit right now with a implementation of that. Let me know your thoughts.